index.cds

```diff
@@ -1,46 +1,54 @@
+
+// The common root-level aspect used in applications like that:
+// using { Attachments } from '@cap-js/attachments'
+aspect Attachments : sap.attachments.Attachments {}
```

Comment:
Essentially, this is the only change. The rest is just using `context` instead of `namespace` and related whitespace changes